doc/source/release/1.15.0-notes.rst

```diff
@@ -17,96 +17,1431 @@ run your code with ``python -Wd`` and check for ``DeprecationWarning`` s).
+* `#20728 <https://github.com/scipy/scipy/issues/20728>`__: BUG: sparse.linalg: Segfault in ``arpack`` with ``ifx``
+* `#20733 <https://github.com/scipy/scipy/issues/20733>`__: DOC: stats.mannwhitneyu: reversed options for ``alternative``...
+* `#20739 <https://github.com/scipy/scipy/issues/20739>`__: DOC/DEV: update commit message guidance
+* `#20740 <https://github.com/scipy/scipy/issues/20740>`__: BLD/DEV: special: build warnings
+* `#20761 <https://github.com/scipy/scipy/issues/20761>`__: ENH: stats.cauchy: improve cdf and quantile accuracy in tail
+* `#20763 <https://github.com/scipy/scipy/issues/20763>`__: BUG: sparse.csgraph, array types: some functions should expect...
+* `#20813 <https://github.com/scipy/scipy/issues/20813>`__: BUG: ``optimize.nnls`` sometimes fails when input ``A`` is a...
+* `#20821 <https://github.com/scipy/scipy/issues/20821>`__: BUG: ``stats.levy_stable.rvs``\ : "S0"-parameterization ignored
+* `#20844 <https://github.com/scipy/scipy/issues/20844>`__: DOC: update testing docs for alternative backends
+* `#20879 <https://github.com/scipy/scipy/issues/20879>`__: MAINT: clean up ``sparse._sputils.getdtype``
+* `#20893 <https://github.com/scipy/scipy/issues/20893>`__: DOC/DEV: Developer docs should mention Accelerate support
+* `#20904 <https://github.com/scipy/scipy/issues/20904>`__: BUG: sparse.csgraph.dijkstra errors on inputs with int64 or no...
+* `#20910 <https://github.com/scipy/scipy/issues/20910>`__: BUG: positional argument ``DeprecationWarning`` message is overly...
+* `#20931 <https://github.com/scipy/scipy/issues/20931>`__: MAINT: Premature setting of attributes in ``HBInfo`` in ``scipy.io._harwell_boei``...
+* `#20957 <https://github.com/scipy/scipy/issues/20957>`__: TST, MAINT: array API GPU test failures
+* `#20963 <https://github.com/scipy/scipy/issues/20963>`__: TST: ``special.tests.test_support_alternative_backends``\ : failure...
+* `#20984 <https://github.com/scipy/scipy/issues/20984>`__: BUG: scipy.optimize.nnls 1.11.4 version has better performance...
+* `#20991 <https://github.com/scipy/scipy/issues/20991>`__: BUG: ``special.pro_rad1`` returns 'nan' for any combination of...
+* `#20994 <https://github.com/scipy/scipy/issues/20994>`__: BUG: ``spatial.distance.cosine`` with ``complex`` arguments raises...
+* `#21009 <https://github.com/scipy/scipy/issues/21009>`__: BUG: Floating point exception when passing the invalid argument...
+* `#21010 <https://github.com/scipy/scipy/issues/21010>`__: BUG: Segmentation fault when passing invalid arguments to some...
+* `#21011 <https://github.com/scipy/scipy/issues/21011>`__: BUG: Double free or corruption when passing invalid arguments...
+* `#21016 <https://github.com/scipy/scipy/issues/21016>`__: BUG: Indexing broken for sparse arrays
+* `#21030 <https://github.com/scipy/scipy/issues/21030>`__: DOC, DEV: release notes ``gh_lists`` touchups
+* `#21044 <https://github.com/scipy/scipy/issues/21044>`__: RFC: quo vadis, ``xp_assert_\*`` infrastructure?
```

Comment:
@tylerjereddy prophetically, at least one bug remains in the `gh_lists` shenanigans :) we should not be escaping asterisks if they occur within code blocks. I can try to look at this before rc1.